ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -443,31 +432,45 @@ export default function BuildQuote({
     fromTokenBalance,
   ]);
 
-  const buildQuotePageLoadedEvent = useNewMetricEvent({
-    event: 'Build Quote Page Loaded',
-    category: 'swaps',
-    sensitiveProperties: {
-      is_hardware_wallet: hardwareWalletUsed,
-      hardware_wallet_type: hardwareWalletType,
-      stx_enabled: smartTransactionsEnabled,
-      current_stx_enabled: currentSmartTransactionsEnabled,
-      stx_user_opt_in: smartTransactionsOptInStatus,
-    },
-  });
-
   useEffect(() => {
     dispatch(resetSwapsPostFetchState());
     dispatch(setReviewSwapClickedTimestamp());
-    buildQuotePageLoadedEvent();
-  }, [dispatch, buildQuotePageLoadedEvent]);
+    metaMetricsEvent({
```

Comment:
[req] rename to trackEvent to match other usages

Reply:
@brad-decker Done [here].([url](https://github.com/MetaMask/metamask-extension/pull/13955/commits/068c92dafb34adb2277d7b9eae602fe9ab9c42b1))